### PR TITLE
Removing Google Analytics 

### PR DESCRIPTION
**Pull request by charleycampbell**
## What does this change?
We want to stop using Google Analytics (GA) to record activity on MMA. This PR removes the use of Google Ananlytics within the manage frontend repo.

## How to test
We will need to merge this change and ask someone with access to the Google Analytics dashboard to check that calls to GA have stopped.

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: client/utilities/hooks/useAnalytics.ts [hunk 1]
```diff
@@ -2,12 +2,6 @@ import { loadScript } from '@guardian/libs';
 import * as Sentry from '@sentry/browser';
 import { useEffect, useState } from 'react';
 import { useLocation } from 'react-router-dom';
-import parse from 'url-parse';
```
Comment (andrewHEguardian) [inline client/utilities/hooks/useAnalytics.ts:5]: does anywhere else use `url-parse` ? maybe we can get rid of it

**Review by andrewHEguardian — APPROVED** [review]
Change looks good to me! 👍

File: client/utilities/hooks/useAnalytics.ts (hunk 1 above)
Comment (charleycampbell) [reply client/utilities/hooks/useAnalytics.ts:5]: great suggestion. I have removed it!

**Comment by andrewHEguardian** [comment]
you should run `yarn` to update the yarn lock file. A way to do this in one step is to remove packages with `yarn remove <packagename>`

**Review by andrewHEguardian — APPROVED** [review]

**Pull request merged**